### ISSUE TITLE
Replace Tests view using similar design as for boots

### ISSUE THREAD

**Issue by gctucker** [kernelci/kernelci-frontend#43]
The current Tests view is hard to follow because it starts from a list of platforms unlike the rest of the front-end which starts with a list of kernel revisions.  There is also the issue of not being able to support groups and sub-groups in test results, so all the test cases appear in flat lists.

Tests view: https://kernelci.org/test/
Test cases: https://kernelci.org/test/board/rk3288-veyron-jaq/job/mainline/kernel/v5.2-3285-gd2b6b4c832f7/

The v4l2-compliance sub-groups all appear at the same level as other test suite groups.  For example, `Required-ioctls` should not appear there but under `v4l2-compliance-uvc`.

It would be good to use the same approach as for builds and boots to show test results starting from a given tree and then choosing a branch / revision / platform / test plan / sub-group / test case.

Sample view from the AGL version which shows results on a per-build basis:
![kernelci-frontend-tests-by-build](https://user-images.githubusercontent.com/3894008/66168700-cb4d3280-e635-11e9-8234-cef107df54a6.png)
https://kernelci.automotivelinux.org/test-build/

**Comment by gctucker** [comment]
This is now a requirement to be able to replace boot tests with the baseline test plan.

The easiest way to solve this issue seems to be to port the AGL fork which has this view:
https://kernelci.automotivelinux.org/test-build/

**Comment by alexandrasp** [comment]
@gctucker can you assign this bug for me?

**Comment by gctucker** [comment]
The source code for the view in the link above can be found on this branch:
https://github.com/BayLibre/kernelci-frontend/tree/wip/tests-by-build

**Comment by gctucker** [comment]
Replaced with roadmap issue https://github.com/kernelci/kernelci-core/issues/307
